src/jl_uv.c

```diff
@@ -27,8 +27,23 @@
 #include <io.h>
 #define write _write
 #endif
-#if defined(_COMPILER_MICROSOFT_) && !defined(_Static_assert)
-#define _Static_assert static_assert
+
+#ifndef static_assert
+#  ifdef __cplusplus
+#    if __cplusplus < 201103L
+#      define static_assert
+#    endif
+#  else
+#    define static_assert
```

Comment:
Shouldn't this be
```C
define static_assert(...)=
```
to effectively remove the assert? Otherwise I'm getting:
```
jl_uv.c: In function 'jl_write':
jl_uv.c:384: warning: left-hand operand of comma expression has no effect
jl_uv.c:382: warning: statement with no effect
```